tests/code_update.py

```diff
@@ -514,16 +575,26 @@ def run(args):
         network.start_and_open(args)
 
         test_verify_quotes(network, args)
+
+        # Measurements
+        test_measurements_tables(network, args)
+        test_add_node_with_bad_code(network, args)
+
+        # Host data/security policy
+        test_host_data_tables(network, args)
+        test_add_node_with_bad_host_data(network, args)
+        test_add_node_with_stubbed_security_policy(network, args)
+
         if snp.IS_SNP:
-            test_snp_measurements_tables(network, args)
-            test_add_node_with_no_uvm_endorsements(network, args)
-            test_host_data_table(network, args)
             test_add_node_without_security_policy(network, args)
-            test_add_node_remove_trusted_security_policy(network, args)
-            test_start_node_with_mismatched_host_data(network, args)
-            test_add_node_with_bad_host_data(network, args)
-        test_add_node_with_bad_code(network, args)
-        # NB: Assumes the current nodes are still using args.package, so must run before test_proposal_invalidation
+            test_add_node_with_bad_security_policy(network, args)
```

Comment:
I'm humming and hahhing about supporting these, but will likely look at it if the PR lives for a while. We don't currently have a way to dynamically set a _different_ security policy for Virtual, and doing so would require some kind of plumbing (probably an env var or file it reads? But still needs to dive through several layers of our infra). The SNP model both has all of this plumbing (with `snp_` specific names), but ignores it for these tests and just fiddles with some files in the security-context directory. Should we duplicate that for virtual? I vote no.

Reply:
+1 for no, some of this can be done by modifying the security policy file, and the collateral.